AK/RefCounted.h

```diff
@@ -44,7 +44,7 @@ class RefCountedBase {
     AK_MAKE_NONMOVABLE(RefCountedBase);
 
 public:
-    using RefCountType = unsigned int;
+    using RefCountType = u32;
```

Comment:
Hmm, hold up, these two are the same size on both 32 and 64 bit, no?

Reply:
Oh, that slipped in randomly from a test. I wasn't going to change the type, hence the alignment fix for `Process`.